packages/round-manager/src/features/api/round.ts

```diff
@@ -70,7 +72,7 @@ export class TransactionBuilder {
  */
 export async function getRoundById(
   signerOrProvider: Web3Provider,
-  roundId: string,
+  roundId: string
```

Comment:
Nothing to change but worth checking if a formatter went rogue anywhere as there are a few "comma diffs" and may cause noise in future reviews.

Reply:
I format everything using prettier with the default config. Prettier --check says that this file is okay, so it's likely that the formatting was not correct previously. and is now correct. Unfortunately, Prettier in default config only inserts trailing commas where es5 supports them, and function parameters with types are not one of those. I'd be happy to enable this option, but I fear it might open a pandora's box of bikeshedding over configuration.

Reply:
I wasn't aware the source file hadn't been previously fed to prettier. Good that it now is!

Reply:
We will do a prettier pass on the whole codebase soon + fail CI if it doesn't pass prettier check. There is a git hook to autoformat staged files too, see README for more info if you'd like.